### PR TITLE
[Mobile] Add basic react native expo sample

### PR DESCRIPTION
**Pull request by YUNQIUGUO**
Update the ort hello world expo sample and add it as a basic react native expo usage app here in inference examples.

### CODE REVIEW

**Comment by YUNQIUGUO** [comment]
Do we want to add the demonstration of how to enable extensions in react native as well in this sample?

File: mobile/examples/React_Native/ort-rn-basic-usage/App.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+import * as ort from 'onnxruntime-react-native';
+import { Asset } from 'expo-asset';
+
+let myModel: ort.InferenceSession;
+
+async function loadModel() {
+  try {
+    // TODO: Can also add example code to demonstrate loading model from bytes here
+    const assets = await Asset.loadAsync(require('./assets/mnist.ort'));
```
Comment (skottmckay) [inline mobile/examples/React_Native/ort-rn-basic-usage/App.tsx:12]: Now that we're using the full package we should use a .onnx file for simplicity. One less step for the user to do. 

Could also tell the user they can view the model in Netron to see that there's one input and one output, and the data types and shapes of those.

File: mobile/examples/React_Native/ort-rn-basic-usage/App.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  } catch (e) {
+    Alert.alert('failed to load model', `${e}`);
+    throw e;
+  }
+}
+
+async function runModel() {
+  try {
+    // Prepare model input data
+    const inputData = new Float32Array(28 * 28);
```
Comment (skottmckay) [inline mobile/examples/React_Native/ort-rn-basic-usage/App.tsx:31]: Add a comment saying in real usage you must set the inputData to the actual input values.

File: mobile/examples/React_Native/ort-rn-basic-usage/README.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+       
+       `metro.config.js` file in this sample application looks like:
+
+       ```js
+       const { getDefaultConfig } = require('@expo/metro-config');
+       const defaultConfig = getDefaultConfig(__dirname);
+       defaultConfig.resolver.assetExts.push('ort');
+       module.exports = defaultConfig;
+       ```
+       Adjust the extension to `.onnx` accordingly if you are loading a ONNX model in your application.
```
Comment (skottmckay) [inline mobile/examples/React_Native/ort-rn-basic-usage/README.md:49]: Update to default to .onnx model. I don't think we need to mention .ORT format models here anymore.

File: mobile/examples/React_Native/ort-rn-basic-usage/README.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+    npm install -g expo-cli
+    ```
+3. Install [Yarn](https://classic.yarnpkg.com/en/docs/install#mac-stable) (Recommended)
+    ```sh
+    npm install -g yarn
+    ```
+**NOTE:**
+   For creating a new react native expo project from scratch, refer to: https://docs.expo.dev/get-started/create-a-project/
+
+## Set up
```
Comment (skottmckay) [inline mobile/examples/React_Native/ort-rn-basic-usage/README.md:19]: The files that are checked in have had all these 'Set up' steps done right? 

If so, this should be called something like 'Steps that were done to add onnxruntime-react-native to the example expo app' so the user can replicate that in their own app.

We should add an item between Prerequisites and this for 'Validate your React Native environment' where we ask them to run the checked in app as-is (maybe move the 'Build and run' info to this new item)
-If the checked-in app fails for us, it's probably an ORT issue
  - ideally we have a CI running the app so we don't have to manually validate this
-If the app fails for them but not us it's probably their RN env. 
-If the app works for them, it's most likely something in their code/setup.

**Comment by skottmckay** [comment]
Is it possible to run the test app in a CI?

**Comment by YUNQIUGUO** [comment]
> Is it possible to run the test app in a CI?

running this in a ci may be possible, requires some additional set up. looking at this: https://docs.expo.dev/build/building-on-ci/
and https://gist.github.com/mrk-han/3d746054eb721a9973817ec7a21363f1 also an example using this `eas-build` in azure devops cicd. will take a look to see if this eas build works locally first.

File: mobile/examples/React_Native/ort-rn-basic-usage/README.md (hunk 4 above)
Comment (YUNQIUGUO) [reply mobile/examples/React_Native/ort-rn-basic-usage/README.md:19]: > The files that are checked in have had all these 'Set up' steps done right?

 The user who clone this app will still have to run `yarn install` and `expo install` the react native npm package. And yes for step `3.` and `4.` are by default done for this sample. put a comment in those steps already.

will see if a CI is possible.

Comment (YUNQIUGUO) [reply mobile/examples/React_Native/ort-rn-basic-usage/README.md]: updated to add a new section as suggested.. hopefully it's clearer now.

**Review by skottmckay — APPROVED** [review]

**Comment by skottmckay** [comment]
If we can add a CI that would be awesome. Can be a separate PR if needed.

**Comment by YUNQIUGUO** [comment]
will add a follow up pr for testing this sample in CI.

**Comment by treksis** [comment]
Hi,

I'm using M2 mac mini
Sonoma 14.0
Using Iphone 15 pro / pro max simulator

The basic react native instruction repository successfully loads the model, but the inference does not work. Here is my repo ```https://github.com/treksis/onnx-react-native-test```

I cloned the repo `https://github.com/microsoft/onnxruntime-inference-examples/tree/main/mobile/examples/React_Native/ort-rn-basic-usage`

1) ```yarn install```
2) ```expo install onnxruntime-react-native@latest```
3) ```expo prebuild```
4) ```expo run:ios```

```
iOS Bundling complete 9352ms
iOS Running app on iPhone 15 Pro
Possible Unhandled Promise Rejection (id: 0):
Error: failed to run model
promiseMethodWrapper@http://localhost:8081/index.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=ai.onnxruntime.example.reactnative.basicusage:5199:45
@http://localhost:8081/index.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=ai.onnxruntime.example.reactnative.basicusage:132778:120
asyncGeneratorStep@http://localhost:8081/index.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=ai.onnxruntime.example.reactnative.basicusage:6094:26
_next@http://localhost:8081/index.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=ai.onnxruntime.example.reactnative.basicusage:6113:29
@http://localhost:8081/index.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=ai.onnxruntime.example.reactnative.basicusage:6118:14
tryCallTwo@http://localhost:8081/index.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=ai.onnxruntime.example.reactnative.basicusage:38422:9
doResolve@http://localhost:8081/index.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=ai.onnxruntime.example.reactnative.basicusage:38561:25
Promise@http://localhost:8081/index.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=ai.onnxruntime.example.reactnative.basicusage:38441:14
@http://localhost:8081/index.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=ai.onnxruntime.example.reactnative.basicusage:6110:25
run@http://localhost:8081/index.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=ai.onnxruntime.example.reactnative.basicusage:132783:28
@http://localhost:8081/index.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=ai.onnxruntime.example.reactnative.basicusage:131861:47
asyncGeneratorStep@http://localhost:8081/index.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=ai.onnxruntime.example.reactnative.basicusage:6094:26
_next@http://localhost:8081/index.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=ai.onnxruntime.example.reactnative.basicusage:6113:29
@http://localhost:8081/index.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=ai.onnxruntime.example.reactnative.basicusage:6118:14
tryCallTwo@http://localhost:8081/index.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=ai.onnxruntime.example.reactnative.basicusage:38422:9
doResolve@http://localhost:8081/index.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=ai.onnxruntime.example.reactnative.basicusage:38561:25
Promise@http://localhost:8081/index.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=ai.onnxruntime.example.reactnative.basicusage:38441:14
@http://localhost:8081/index.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=ai.onnxruntime.example.reactnative.basicusage:6110:25
@http://localhost:8081/index.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=ai.onnxruntime.example.reactnative.basicusage:131203:40
asyncGeneratorStep@http://localhost:8081/index.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=ai.onnxruntime.example.reactnative.basicusage:6094:26
_next@http://localhost:8081/index.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=ai.onnxruntime.example.reactnative.basicusage:6113:29
@http://localhost:8081/index.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=ai.onnxruntime.example.reactnative.basicusage:6118:14
tryCallTwo@http://localhost:8081/index.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=ai.onnxruntime.example.reactnative.basicusage:38422:9
doResolve@http://localhost:8081/index.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=ai.onnxruntime.example.reactnative.basicusage:38561:25
Promise@http://localhost:8081/index.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=ai.onnxruntime.example.reactnative.basicusage:38441:14
@http://localhost:8081/index.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=ai.onnxruntime.example.reactnative.basicusage:6110:25
_runModel@http://localhost:8081/index.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=ai.onnxruntime.example.reactnative.basicusage:131216:27
runModel@http://localhost:8081/index.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=ai.onnxruntime.example.reactnative.basicusage:131192:27
_performTransitionSideEffects@http://localhost:8081/index.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=ai.onnxruntime.example.reactnative.basicusage:82522:22
_receiveSignal@http://localhost:8081/index.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=ai.onnxruntime.example.reactnative.basicusage:82472:45
onResponderRelease@http://localhost:8081/index.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=ai.onnxruntime.example.reactnative.basicusage:82328:34
invokeGuardedCallbackProd@http://localhost:8081/index.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=ai.onnxruntime.example.reactnative.basicusage:83044:21
invokeGuardedCallback@http://localhost:8081/index.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=ai.onnxruntime.example.reactnative.basicusage:83220:42
invokeGuardedCallbackAndCatchFirstError@http://localhost:8081/index.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=ai.onnxruntime.example.reactnative.basicusage:83234:36
executeDispatch@http://localhost:8081/index.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=ai.onnxruntime.example.reactnative.basicusage:83311:48
executeDispatchesInOrder@http://localhost:8081/index.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=ai.onnxruntime.example.reactnative.basicusage:83333:26
executeDispatchesAndRelease@http://localhost:8081/index.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=ai.onnxruntime.example.reactnative.basicusage:85106:35
forEach@[native code]
forEachAccumulated@http://localhost:8081/index.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=ai.onnxruntime.example.reactnative.basicusage:83927:22
runEventsInBatch@http://localhost:8081/index.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=ai.onnxruntime.example.reactnative.basicusage:85126:27
runExtractedPluginEventsInBatch@http://localhost:8081/index.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=ai.onnxruntime.example.reactnative.basicusage:85236:25
@http://localhost:8081/index.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=ai.onnxruntime.example.reactnative.basicusage:85207:42
batchedUpdates$1@http://localhost:8081/index.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=ai.onnxruntime.example.reactnative.basicusage:98672:20
batchedUpdates@http://localhost:8081/index.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=ai.onnxruntime.example.reactnative.basicusage:85082:36
_receiveRootNodeIDEvent@http://localhost:8081/index.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=ai.onnxruntime.example.reactnative.basicusage:85206:23
receiveTouches@http://localhost:8081/index.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=ai.onnxruntime.example.reactnative.basicusage:85295:34
__callFunction@http://localhost:8081/index.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=ai.onnxruntime.example.reactnative.basicusage:3677:36
@http://localhost:8081/index.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=ai.onnxruntime.example.reactnative.basicusage:3397:31
__guard@http://localhost:8081/index.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=ai.onnxruntime.example.reactnative.basicusage:3614:15
callFunctionReturnFlushedQueue@http://localhost:8081/index.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=ai.onnxruntime.example.reactnative.basicusage:3396:21
```



I also tried to make from the scratch with the instruction, it loads the model but the inference does not work.
1) ```npx create-expo-app my-app```
2) ```npm install onnxruntime-react-native```
3) To app.json, add ```"plugins": ["onnxruntime-react-native"]``` 
4) Create ```metro.config.js```

```
const { getDefaultConfig } = require('@expo/metro-config');
const defaultConfig = getDefaultConfig(__dirname);
defaultConfig.resolver.assetExts.push('onnx');
module.exports = defaultConfig;
```

5) To build ```expo prebuild```
6) To run ```expo run:ios```


```
iOS Bundling complete 2074ms
iOS Bundling complete 39ms
iOS Running app on iPhone 15 Pro
Possible Unhandled Promise Rejection (id: 0):
Error: failed to run model
Error: failed to run model
    at promiseMethodWrapper (http://localhost:8081/node_modules/expo/AppEntry.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=com.treksis.myapp:5196:45)
    at ?anon_0_ (http://localhost:8081/node_modules/expo/AppEntry.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=com.treksis.myapp:131664:120)
    at next (native)
    at asyncGeneratorStep (http://localhost:8081/node_modules/expo/AppEntry.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=com.treksis.myapp:6087:26)
    at _next (http://localhost:8081/node_modules/expo/AppEntry.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=com.treksis.myapp:6106:29)
    at anonymous (http://localhost:8081/node_modules/expo/AppEntry.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=com.treksis.myapp:6111:14)
    at tryCallTwo (/Users/distiller/react-native/packages/react-native/sdks/hermes/build_iphonesimulator/lib/InternalBytecode/InternalBytecode.js:61:9)
    at doResolve (/Users/distiller/react-native/packages/react-native/sdks/hermes/build_iphonesimulator/lib/InternalBytecode/InternalBytecode.js:216:25)
    at Promise (/Users/distiller/react-native/packages/react-native/sdks/hermes/build_iphonesimulator/lib/InternalBytecode/InternalBytecode.js:82:14)
    at anonymous (http://localhost:8081/node_modules/expo/AppEntry.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=com.treksis.myapp:6103:25)
    at apply (native)
    at run (http://localhost:8081/node_modules/expo/AppEntry.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=com.treksis.myapp:131669:28)
    at ?anon_0_ (http://localhost:8081/node_modules/expo/AppEntry.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=com.treksis.myapp:130747:47)
    at next (native)
    at asyncGeneratorStep (http://localhost:8081/node_modules/expo/AppEntry.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=com.treksis.myapp:6087:26)
    at _next (http://localhost:8081/node_modules/expo/AppEntry.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=com.treksis.myapp:6106:29)
    at anonymous (http://localhost:8081/node_modules/expo/AppEntry.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=com.treksis.myapp:6111:14)
    at tryCallTwo (/Users/distiller/react-native/packages/react-native/sdks/hermes/build_iphonesimulator/lib/InternalBytecode/InternalBytecode.js:61:9)
    at doResolve (/Users/distiller/react-native/packages/react-native/sdks/hermes/build_iphonesimulator/lib/InternalBytecode/InternalBytecode.js:216:25)
    at Promise (/Users/distiller/react-native/packages/react-native/sdks/hermes/build_iphonesimulator/lib/InternalBytecode/InternalBytecode.js:82:14)
    at anonymous (http://localhost:8081/node_modules/expo/AppEntry.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=com.treksis.myapp:6103:25)
    at apply (native)
    at run (http://localhost:8081/node_modules/expo/AppEntry.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=com.treksis.myapp:130757:28)
    at ?anon_0_ (http://localhost:8081/node_modules/expo/AppEntry.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=com.treksis.myapp:130093:40)
    at next (native)
    at asyncGeneratorStep (http://localhost:8081/node_modules/expo/AppEntry.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=com.treksis.myapp:6087:26)
    at _next (http://localhost:8081/node_modules/expo/AppEntry.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=com.treksis.myapp:6106:29)
    at anonymous (http://localhost:8081/node_modules/expo/AppEntry.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=com.treksis.myapp:6111:14)
    at tryCallTwo (/Users/distiller/react-native/packages/react-native/sdks/hermes/build_iphonesimulator/lib/InternalBytecode/InternalBytecode.js:61:9)
    at doResolve (/Users/distiller/react-native/packages/react-native/sdks/hermes/build_iphonesimulator/lib/InternalBytecode/InternalBytecode.js:216:25)
    at Promise (/Users/distiller/react-native/packages/react-native/sdks/hermes/build_iphonesimulator/lib/InternalBytecode/InternalBytecode.js:82:14)
    at anonymous (http://localhost:8081/node_modules/expo/AppEntry.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=com.treksis.myapp:6103:25)
    at apply (native)
    at _runModel (http://localhost:8081/node_modules/expo/AppEntry.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=com.treksis.myapp:130105:27)
    at apply (native)
    at runModel (http://localhost:8081/node_modules/expo/AppEntry.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=com.treksis.myapp:130085:27)
    at _performTransitionSideEffects (http://localhost:8081/node_modules/expo/AppEntry.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=com.treksis.myapp:81860:22)
    at _receiveSignal (http://localhost:8081/node_modules/expo/AppEntry.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=com.treksis.myapp:81810:45)
    at onResponderRelease (http://localhost:8081/node_modules/expo/AppEntry.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=com.treksis.myapp:81666:34)
    at apply (native)
    at invokeGuardedCallbackProd (http://localhost:8081/node_modules/expo/AppEntry.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=com.treksis.myapp:82376:21)
    at apply (native)
    at invokeGuardedCallback (http://localhost:8081/node_modules/expo/AppEntry.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=com.treksis.myapp:82552:42)
    at apply (native)
    at invokeGuardedCallbackAndCatchFirstError (http://localhost:8081/node_modules/expo/AppEntry.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=com.treksis.myapp:82566:36)
    at executeDispatch (http://localhost:8081/node_modules/expo/AppEntry.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=com.treksis.myapp:82643:48)
    at executeDispatchesInOrder (http://localhost:8081/node_modules/expo/AppEntry.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=com.treksis.myapp:82665:26)
    at executeDispatchesAndRelease (http://localhost:8081/node_modules/expo/AppEntry.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=com.treksis.myapp:84438:35)
    at executeDispatchesAndReleaseTopLevel (http://localhost:8081/node_modules/expo/AppEntry.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=com.treksis.myapp:84445:43)
    at forEach (native)
    at forEachAccumulated (http://localhost:8081/node_modules/expo/AppEntry.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=com.treksis.myapp:83259:22)
    at runEventsInBatch (http://localhost:8081/node_modules/expo/AppEntry.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=com.treksis.myapp:84458:27)
    at runExtractedPluginEventsInBatch (http://localhost:8081/node_modules/expo/AppEntry.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=com.treksis.myapp:84568:25)
    at anonymous (http://localhost:8081/node_modules/expo/AppEntry.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=com.treksis.myapp:84539:42)
    at batchedUpdates$1 (http://localhost:8081/node_modules/expo/AppEntry.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=com.treksis.myapp:98004:20)
    at batchedUpdates (http://localhost:8081/node_modules/expo/AppEntry.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=com.treksis.myapp:84414:36)
    at _receiveRootNodeIDEvent (http://localhost:8081/node_modules/expo/AppEntry.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=com.treksis.myapp:84538:23)
    at receiveTouches (http://localhost:8081/node_modules/expo/AppEntry.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=com.treksis.myapp:84627:34)
    at apply (native)
    at __callFunction (http://localhost:8081/node_modules/expo/AppEntry.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=com.treksis.myapp:3679:36)
    at anonymous (http://localhost:8081/node_modules/expo/AppEntry.bundle//&platform=ios&dev=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&app=com.treksis.myapp:3399:31)
    at __guard (http://localhost:8081/node_modules/expo/AppEntry.bundle//&platform=ios&dev=true&minify=false&inlineSou...(truncated to the first 10000 characters)
```

Screenshots

image

image

I need help

Thank you

**Comment by YUNQIUGUO** [comment]
Hi, 

If you are running the check-in app as is (and running the mnist.onnx model),  it currently requires a fix in the input tensor shape value. See this pr: https://github.com/microsoft/onnxruntime-inference-examples/pull/321

After the fix checked-in, should be able to run the app.

Also, side question: have you tried running in Android end as well?

**Comment by treksis** [comment]
Changing one line solved the problem. Jesus... Thank you so much!!!

```
https://github.com/microsoft/onnxruntime-inference-examples/pull/321/commits/8ffda86732ced80417627509e2f4d02802790031
```
Closed

**Comment by treksis** [comment]
Of course it is working on Android...! Thank you

image


> Hi,
> 
> If you are running the check-in app as is (and running the mnist.onnx model), it currently requires a fix in the input tensor shape value. See this pr: #321
> 
> After the fix checked-in, should be able to run the app. Sorry for the inconvience.
> 
> Also, side question: have you tried running in Android end as well?

**Pull request merged**